This patch allows for turning on center button led to blink in screen/lock menu when notifications are present. -Jack87
--- /usr/palm/applications/com.palm.app.screenlock/app/controllers/securityconfig-assistant.js.orig	Mon Sep 28 16:23:58 2009
+++ /usr/palm/applications/com.palm.app.screenlock/app/controllers/securityconfig-assistant.js	Mon Sep 28 16:25:03 2009
@@ -51,8 +51,8 @@
 		this.controller.setupWidget('showAlerts', this.onOffToggleOpt, this.alertToggleModel);
 		Mojo.Event.listen($('showAlerts'),'mojo-property-change', this.toggleShowAlerts.bindAsEventListener(this));
 		
-		//this.controller.setupWidget('blinkAlerts', this.onOffToggleOpt, this.ledThrobberToggleModel);
-		//Mojo.Event.listen($('blinkAlerts'),'mojo-property-change', this.toggleLEDThrobber.bindAsEventListener(this));
+		this.controller.setupWidget('blinkAlerts', this.onOffToggleOpt, this.ledThrobberToggleModel);
+		Mojo.Event.listen($('blinkAlerts'),'mojo-property-change', this.toggleLEDThrobber.bindAsEventListener(this));
 		
 		this.controller.setupWidget('switchApps', this.onOffToggleOpt, this.switchApptoggleModel);
 		Mojo.Event.listen($('switchApps'),'mojo-property-change', this.toggleSwitchApps.bindAsEventListener(this));
@@ -245,10 +245,10 @@
 			}
 		}
 		
-		/*if (payload.LEDThrobberEnabled != undefined) {
+		if (payload.LEDThrobberEnabled != undefined) {
 			this.ledThrobberToggleModel.value = payload.LEDThrobberEnabled;
 			this.controller.modelChanged(this.ledThrobberToggleModel, this);
-		}*/
+		}
 		
 		//Demo Device -- Hide the PIN / Password options.
 		if(payload.onDeviceDemoRunning != undefined && payload.onDeviceDemoRunning === true) {
--- /usr/palm/applications/com.palm.app.screenlock/app/views/securityconfig/securityconfig-scene.html.orig	Mon Sep 28 16:23:13 2009
+++ /usr/palm/applications/com.palm.app.screenlock/app/views/securityconfig/securityconfig-scene.html	Mon Sep 28 16:23:42 2009
@@ -89,17 +89,17 @@
 					<div class="title capitalize" x-mojo-loc=''>Show when locked</div>
 				</div>
 			</div>
-			<!--<div class="palm-row last">
+			<div class="palm-row last">
 				<div class="palm-row-wrapper">
 					<div x-mojo-element="ToggleButton" id="blinkAlerts"></div>
 					<div class="title capitalize" x-mojo-loc=''>Blink notifications</div>
 				</div>
-			</div>-->
+			</div>
 		</div>
 	</div>
 
-	<!--<div class="palm-info-text single" x-mojo-loc=''>
+	<div class="palm-info-text single" x-mojo-loc=''>
 		The gesture area blinks when new notifications arrive.
-	</div>-->
+	</div>
 
 </div>
\ No newline at end of file

